lightning/src/ln/outbound_payment.rs

```diff
@@ -226,9 +226,9 @@ impl PendingOutboundPayment {
 pub enum Retry {
 	/// Max number of attempts to retry payment.
 	///
-	/// This is the number of additional paths along which the payment will be retried after the
-	/// first payment. If two paths fail at around the same time and are retried with a single
-	/// MPP path/HTLC, it will only count as a single retry.
+	/// This is the number of retried sends done. Each send may be multiple HTLCs along multiple
+	/// paths if the router decides to split up a send, and may retry multiple failed HTLCs if they
+	/// failed around the same time and were retried with a single [`Route`].
```

Comment:
Not sure I understand why "and were retried with a single [`Route`]" is included here.